### PR TITLE
Game board

### PR DESCRIPTION
**Pull request by JimD325**

### CODE REVIEW

File: src/components/GameBoard/GameBoard.css [hunk 1]
```diff
@@ -26,19 +26,48 @@
...
+
+#damage {
   filter: blur(5px); 
   transition: filter .1s linear;
   animation: flash 0.1s linear;
 }
 
+#death {
+  /* transition: .5s linear;
+  transform: rotate(-.25turn); */
```
Comment (laptopadventure) [inline src/components/GameBoard/GameBoard.css:43]: commented out code

File: src/components/GameBoard/GameBoard.css [hunk 2]
```diff
@@ -26,19 +26,48 @@
 
 .raccoon {
   position: absolute;
-  transition: margin 0.5s ease-in-out;
+  /* transition: margin 0.5s ease-in-out; */
```
Comment (laptopadventure) [inline src/components/GameBoard/GameBoard.css:29]: commented code

File: src/components/GameBoard/GameBoard.tsx [hunk 3]
```diff
@@ -142,44 +155,42 @@ const Board = () => {
...
-
-
 // TODO Conditionally render Pause/play button depending on whether or not the game is playing. 
 const Buttons = (props: any) => {
   const { setScreen } = useContext(ScreenContext)
   const { startRound } = props;
   const { winWidth, winHeight } = useWindowDimensions()
   if (winWidth > winHeight) {
-    console.log("wide")
+    // console.log("wide")
```
Comment (laptopadventure) [inline src/components/GameBoard/GameBoard.tsx:193]: commented code

**Review by laptopadventure — CHANGES_REQUESTED** [review]

File: src/components/GameBoard/GameBoard.tsx (hunk 3 above)
Comment (ZayahPapaya) [reply src/components/GameBoard/GameBoard.tsx:193]: wide

**Review by laptopadventure — APPROVED** [review]

**Pull request merged**